src/main/java/mat/dto/MeasureTransferDTO.java

```diff
@@ -0,0 +1,53 @@
+package mat.dto;
+
+import mat.client.measure.ManageMeasureDetailModel;
+
+import java.io.Serializable;
+
+public class MeasureTransferDTO implements Serializable {
+    private String emailId;
+    private String harpId;
+    private ManageMeasureDetailModel manageMeasureDetailModel;
+    private String measureResourceJson;
+    private String libraryResourcesJson;
+
+    public String getEmailId() {
```

Comment:
Just curious, lombok is not available?

Reply:
@sb-cecilialiu, that is a good point. I thought MAT doesn't have it but turns out it has. I'll replace it with Lombok annotations. Thank you!